fsm.cpp

```diff
@@ -9,9 +9,16 @@
 fsm::stateHandler_t fsm::stateHandler = fsm::state0_idle;
 unsigned int lastStateHandleTime = millis();
 
-static unsigned int COUNTDOWN_TIMER = 15000;
-static unsigned int DURATION_TIMER = 30000;
-static unsigned int STILLNESS_TIMER = 15000;
+const unsigned int MAX_COUNTDOWN_TIMER = 20000; // 20s
+const unsigned int MIN_DURATION_TIMER = 30000; // 30s
+const unsigned int MIN_STILLNESS_TIMER = 30000; // 30s
+const unsigned int DEFAULT_COUNTDOWN_TIMER = 15000; // 15s
+const unsigned int DEFAULT_DURATION_TIMER = 30000; // 30s
+const unsigned int DEFAULT_STILLNESS_TIMER = 15000; // 15s
+
+static unsigned int COUNTDOWN_TIMER = DEFAULT_COUNTDOWN_TIMER;
+static unsigned int DURATION_TIMER = DEFAULT_DURATION_TIMER;
+static unsigned int STILLNESS_TIMER = DEFAULT_STILLNESS_TIMER;
```

Comment:
Generally speaking, only `const` values are in all capital letters. Could these be renamed to make it clear that they change? Thanks!

Reply:
I have changed them. On the topic of this, I've also changed the state names to use camel case for consistency.